src/components/Breadcrumb/Breadcrumbs.tsx

```diff
@@ -0,0 +1,23 @@
+import { makeStyles, Breadcrumbs as MUIBreadcrumbs } from '@material-ui/core';
+import { FC } from 'react';
+import * as React from 'react';
+
+const useStyles = makeStyles(({ palette }) => ({
+  // Not sure about doing it this way. Just getting rid of styling on last a element to make it not a link
+  breadcrumb: {
+    '& .MuiBreadcrumbs-li:last-child a': {
+      pointerEvents: 'none',
+      cursor: 'default',
+      textDecoration: 'none',
+      color: palette.text.primary,
+    },
+  },
+}));
+
+export const Breadcrumbs: FC = ({ children }) => {
```

Comment:
We should add a Props interface here, and it should extend the material-ui `BreadcrumbProps` interface. e.g.:

Import the props interface:
```typescript
import { BreadcrumbsProps as MUIBreadcrumbsProps } from '@material-ui/core';
```

Define props for this component:
```typescript
export interface BreadcrumbsProps extends MUIBreadcrumbsProps {
  children: ReactNode;
}
```

Defining the component:
```typescript
export const Breadcrumbs: FC<BreadcrumbsProps> = ({ children, ...restProps }) => {
```

Reply:
I changed it to take in rest props and also have the interface